packages/suite/src/components/firmware/Button.tsx

```diff
@@ -1,88 +1,32 @@
 import React from 'react';
 
-import { Tooltip, Button } from '@trezor/components';
+import { Button, ButtonProps } from '@trezor/components';
```

Comment:
I don't think this file is relevant anymore. We are not doing anything else but wrapping a translation and spreading incoming props to a Button component. Could be done directly where they are used.

Reply:
Hmm, true it does not do much but it helps me a lot to stay uniform as these buttons are used on multiple places, definitely not one. Not to mention that it makes refactoring easier when product decides that they want to change something.